Supplier access

andreo141 opened the pull request:
This PR mocks a new role for suppliers.
This also includes a migration to add supplier Vanden Broele to the mock users.
Currently waiting on acmidm role, but this is just to test how it would work when we get the correct role.

**Frontend::** https://github.com/lblod/frontend-openproceshuis/compare/feature/supplier-access
**Jira:** https://binnenland.atlassian.net/browse/OPH-499

MartijnBogaert commented on config/authorization/config.lisp:
```diff
@@ -125,6 +140,10 @@
        :to-graph organizations
        :for-allowed-group "organization-processes-editor")
 
+(grant (read write)
+       :to-graph organizations
+       :for-allowed-group "supplier")
```
In addition to this, you'll also have to provide read write access to the shared graph. This makes sure process data is written to a "private" organizational graph, as well as the "public" shared graph. If the data is not present in the shared graph, other users won't be able to see it.

MartijnBogaert reviewed (commented):
Looks very nice! I added 1 comment because I think you missed something.
Also, I think we can make this even more 'compact', I have a commit ready.

MartijnBogaert commented:
@andreo141 I added a commit to make the auth config more compact. Since suppliers (currently) don't have any extra functionality compared to 'normal' organizations, they can simply be regarded as such. So I got rid of their own `suppliers` group and just added a check for their role to the `organization-processes-editor` and `shared-processes-editor` groups. What do you think?

andreo141 commented:
> @andreo141 I added a commit to make the auth config more compact. Since suppliers (currently) don't have any extra functionality compared to 'normal' organizations, they can simply be regarded as such. So I got rid of their own `suppliers` group and just added a check for their role to the `organization-processes-editor` and `shared-processes-editor` groups. What do you think?

That's indeed a cleaner approach. Let's go with that :)
